draft-nygren-httpbis-httpssvc.md

```diff
@@ -524,9 +524,9 @@ origin hostname's CNAME chain if possible.  This will ensure that the required
 address records are already present in the client's DNS cache.
 
 Highly performance-sensitive clients MAY implement the following special-
-case shortcut to avoid increased connection time: if (1) all of the
-HTTPSSVC records returned have SvcRecordType = 0, (2) none of their
-SvcDomainNames are in the DNS cache, and (3) the address queries for the
+case shortcut to avoid increased connection time: if (1) one of the
+HTTPSSVC records returned has SvcRecordType = 0, (2) its SvcDomainName
+is not in the DNS cache, and (3) the address queries for the
```

Comment:
Quibble:  This could be read as saying that in the presence of multiple "0"s, you MAY ignore them all if any of them aren't in your cache.  But then again, you can ignore them entirely for any reason you like, so it's kind of a moot point.

Reply:
I hit merge, but feel free to suggest improved text (or logic!) here.